src/main/SnakeAndLadders/Practice/Model/Ladders.java

```diff
@@ -0,0 +1,27 @@
+package SnakeAndLadders.Practice.Model;
+
+public class Ladders {
```

Comment:
I think this entity is for a single ladder and should be named Ladder instead of Ladders